examples/index.html

```diff
@@ -93,6 +93,20 @@ <h4 class="card-header">
           </a>
         </div>
       </div>
+      <div class="col-sm-6">
+        <div class="card-deck">
+          <a class="card" href="/dominantspeaker" style="text-decoration: none">
+            <h4 class="card-header">
+              Dominant Speaker
+            </h4>
+            <div class="card-block">
+              <p class="card-text">
+                This app demonstrates the dominantSpeaker property, which can be used to identify current Speaker
```

Comment:
@makarandp0 

> This app demonstrates the Dominant Speaker API for Group or Small Group Rooms.
In order to run this example, please go to the [Programmable Video Settings](https://www.twilio.com/console/video/configure) in the
Twilio Console and select the **Room Type** as **Group** or **Group-Small**.